### ISSUE TITLE
tests don't pass out-of-the-box

### ISSUE THREAD

**Issue by IBwWG** [benawad/feathersjs-menu-monkey-backend#2]
`npm test` has the linter complain with a bunch of 'must use singlequote' messages (and one 'missing semicolon') about `src/services/recipe/hooks/process.js`.

`npm run mocha` fails seemingly due to a typo (time.test.js is under 'recipes' and looks for 'recipes', but under `src/services` the directory is called 'recipe,' singular.)  I saw this in one of your videos (great series btw) where the cause of node not starting up was a plural in one place, singular in another.  Which is the 'right' standard to use?

**Comment by IBwWG** [comment]
In fact I notice that `test/services` contains both 'recipe' and 'recipes'.  Is this intentional?

**Comment by IBwWG** [comment]
Oh, actually I guess it's not a typo, even under `src/services/recipe/hooks/` there is no `time.js`...

**Comment by IBwWG** [comment]
With that one aside, another test also fails: `test/services/recipe/hooks/process.test.js` gets `Error: Invalid json`.  Other than that, 5 passing.

**Comment by benawad** [comment]
Thanks for bringing this to my attention.

I went ahead and fixed the things the linter was complaining about.

All the tests were auto-generated by Feathersjs, so some of them don't fit the application. I removed the ones that are unneeded and corrected the ones with errors.

The standard naming convention I have seen is to use the singular form when creating the service (like recipe), and then use the plural when referring to it in the client or in the url. So for example, you would make a POST request to localhost:3030/recipes **NOT** localhost:3030/recipe.
